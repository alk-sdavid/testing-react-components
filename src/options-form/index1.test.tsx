import { render } from '@testing-library/react';
import { OptionsForm } from '.';

describe('<OptionsForm />', () => {
  it('should be well rendered', () => {
    const { container } = render(<OptionsForm sendNotification={() => {}} />);
    expect(container).toMatchSnapshot();
  });

  it('should be still well rendered', () => {
    const { container } = render(
      <OptionsForm sendNotification={() => {}} canUpdate={false} />,
    );
    expect(container).toMatchSnapshot();
  });
});
