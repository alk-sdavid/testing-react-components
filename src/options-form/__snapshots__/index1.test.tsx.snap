// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<OptionsForm /> should be still well rendered 1`] = `
<div>
  <div>
    Fetching options...
  </div>
</div>
`;

exports[`<OptionsForm /> should be well rendered 1`] = `
<div>
  <div>
    Fetching options...
  </div>
</div>
`;
